tcp-echo.go

```diff
@@ -45,6 +52,44 @@ func main() {
 		message = message + fmt.Sprintf("Service %s.\n", serviceAccountName)
 	}
 
+	if tlsPort != "" && tlsCertFile != "" && tlsKeyFile != "" {
+
+		roots := x509.NewCertPool()
+		if tlsCACertFile != "" {
+			caCertPEM, err := os.ReadFile(tlsCACertFile)
+			if err != nil {
+				log.Panicf("failed to load CA, err %v", err)
+			}
+			roots.AppendCertsFromPEM(caCertPEM)
```

Comment:
Check the return value of `AppendCertsFromPEM`? It may fail to parse the cert

Reply:
done.